modules/test/playwright/pages/object-web/ModelBuilderPage.ts

```diff
@@ -69,10 +69,9 @@ export class ModelBuilderPage {
 		this.leftSidebarItems = page.locator(
 			'li.treeview-item div.autofit-row'
 		);
-		this.modalDeleteObjectDefinitionConfirmationButton = page.getByRole(
-			'button',
-			{exact: true, name: 'Delete'}
-		);
+		this.modalDeleteObjectDefinitionConfirmationButton = page
+			.locator('div.modal')
+			.getByRole('button', {exact: true, name: 'Delete'});
```

Comment:
Hi @igor-franca ! :smile: 
Just explaining this change better, I thought it would be better to transform this locator into something that works for any delete button within a modal, I didn't find any other way than locating it through the class with the word "modal".
If there is another better way, let me know!

ps: thank you for your patience! :heart_decoration:

Reply:
Hi @rafaellajordao, you could try using `getByRole='dialog'` to identify when the button is in the modal?

Let me know if its works!

Reply:
Hi @rafaellajordao, you could try using getByRole='dialog' to identify when the button is in the modal?

Let me know if its works!